packages/esm-patient-chart-app/src/visit/visits-widget/visit-context/visit-context.ts

```diff
@@ -0,0 +1,12 @@
+import { getVisitStore, useStoreWithActions, type Visit } from '@openmrs/esm-framework';
+
+export function useVisitContextStore() {
+  return useStoreWithActions(getVisitStore(), {
```

Comment:
Shouldn't this be moved into the react utils? Also, isn't this indirectly `useVisit`?